tasks/broad/ImputationTasks.wdl

```diff
@@ -560,9 +560,9 @@ task SelectVariantsByIds {
 
```

Comment:
We can probably move this task definition to the VerifyImputation wdl

Reply:
I moved `CrosscheckFingerprints` into `VerifyImputation.wdl`, but `SplitMultiSampleVcf` is referenced in `Imputation.wdl` so I left it. Which task are you referring to here?